test/Microsoft.AspNet.Mvc.TagHelpers.Test/TextAreaTagHelperTest.cs

```diff
@@ -46,32 +47,32 @@ public class TextAreaTagHelperTest
                 {
                     { null, typeof(Model), null,
                         new NameAndId("Text", "Text"),
-                        Environment.NewLine + "HtmlEncode[[]]" },
+                        Environment.NewLine },
```

Comment:
This encoding changed because <a href="https://github.com/aspnet/HttpAbstractions/blob/dev/src/Microsoft.Framework.WebEncoders.Core/EncoderExtensions.cs#L28">WebEncoders' extension method</a> skips null <b>and</b> empty strings. This was not the case before because we encoded the string separately using <a href="https://github.com/aspnet/Testing/blob/dev/src/Microsoft.Framework.WebEncoders.Testing/CommonTestEncoder.cs#L88">CommonTestEncoder</a> (which returns a `HtmlEncode[[]]` when encoding empty strings) and we then write this to the writer.